### ISSUE TITLE
Firmware update page: text is too wordy

### ISSUE THREAD

**Issue by obra** [keyboardio/Chrysalis#222]
> Updating the firmware

I don't think we need this heading?

> Updating the firmware is a safe process, it's very hard to brick your keyboard even with bad firmware, as most keyboards provide a way to go stay in bootloader mode, where new firmware can be flashed. Nevertheless, updating the firmware will overwrite the previous one. If you customised your firmware, make sure you're flashing one that you are comfortable with. If you wish to proceed, please consult the documentation of your keyboard to see how to enable uploading new firmware, and then press the Update button to continue.

>To see what is included in the default firmware, please see the Chrysalis-Firmware-Bundle repository.

Updating or "flashing" your keyboard's firmware is how we teach it new tricks. Chrysalis will install a new version of your keyboard's firmware which includes support for customizing the key layout, as well as other features.

If you've previously customized your keyboard's firmware, this will overwrite your custom firmware. You can always find the source code of the firmware Chrysalis is installing here: <url>

--Device specific text goes here--

>Once the upload is finished - either successfully or with errors -, you will be taken back to the initial keyboard selection screen. This is normal.

Once the upload is done, Chrysalis will take you back to the keyboard selection screen.

**Comment by algernon** [comment]
Liking the new text, thank you!
